src/sqlfluff/utils/reflow/helpers.py

```diff
@@ -5,7 +5,8 @@
 from typing import Iterable, List
 
 from sqlfluff.core.parser import BaseSegment, RawSegment
-from sqlfluff.core.rules.base import LintFix, LintResult
+from sqlfluff.core.rules import LintFix
+from sqlfluff.core.rules.base import LintResult
```

Comment:
```suggestion
from sqlfluff.core.rules import LintFix, LintResult
```

Same comment as above...